doc/src/language.md

```diff
@@ -241,25 +241,3 @@ iff
 storage
    a.x => z
 ```
-
```

Comment:
This is a section in the docs that was removed by a PR because we no longer have this constraint. Then it was readded by a subsequent PR because of a merge error.